custom/icds_reports/reports/governance_apis.py

```diff
@@ -121,3 +121,47 @@ def base_data(row_data):
 def get_state_names():
     return list(AwcLocation.objects.filter(aggregation_level=AggregationLevels.STATE, state_is_test=0
                                            ).values('state_site_code', 'state_name'))
+
+
+@icds_quickcache(['length', 'year', 'month', 'order', 'query_filters'], timeout=30 * 60)
+def get_cbe_data(length, year, month, order, query_filters):
+    data = AggGovernanceDashboard.objects.filter(
+        month=date(year, month, 1),
+        **query_filters
+    ).order_by(*order).annotate(
+        cbe_conducted_1=Case(
+            When(cbe_date_1__isnull=False, then=Value('yes')),
```

Comment:
why use a string instead of just a boolean value

Reply:
That is how we defined in the api.